compositor_chromium/src/v8/object.rs

```diff
@@ -0,0 +1,49 @@
+use crate::{cef_string::CefString, validated::Validated};
+
+use super::value::{V8Value, V8ValueError};
+
+pub struct V8Object(pub(super) Validated<chromium_sys::cef_v8value_t>);
+
+impl V8Object {
+    pub fn has(&self, key: &str) -> Result<bool, V8ValueError> {
+        let inner = self.0.get()?;
+        let key = CefString::new_raw(key);
+        unsafe {
+            let has_value = (*inner).has_value_bykey.unwrap();
+            Ok(has_value(inner, &key) == 1)
+        }
+    }
+
+    /// Returns `true` if value was set successfully
+    pub fn set(
+        &mut self,
+        key: &str,
+        value: &V8Value,
+        attribute: V8PropertyAttribute,
+    ) -> Result<bool, V8ValueError> {
+        let inner = self.0.get()?;
+        let key = CefString::new_raw(key);
+        unsafe {
+            let set_value = (*inner).set_value_bykey.unwrap();
+            Ok(set_value(inner, &key, value.get_raw()?, attribute as u32) == 1)
+        }
+    }
+
+    /// Returns `true` if value was deleted successfully
+    pub fn delete(&mut self, key: &str) -> Result<bool, V8ValueError> {
```

Comment:
Since we return the result here, shouldn't unsuccessful value deletion be `Err`? 
Then this function typing would be `Result<(), DeletionError>`

Reply:
Yeah, good idea